src/main/java/net/rptools/maptool/client/AppActions.java

```diff
@@ -2870,6 +2870,8 @@ protected void executeAction() {
                   MapTool.showError("msg.error.failedSaveMap", ioe);
                 }
               }
+            } else {
```

Comment:
I think it would be better to just change "if (chooser.showSaveDialog(MapTool.getFrame()) == JFileChooser.APPROVE_OPTION) { do save }" to "if (chooser.showSaveDialog(MapTool.getFrame()) != JFileChooser.APPROVE_OPTION) break;" instead.

Reply:
suggestions completed